core-ui/src/shared/components/StatsGraph/StatsGraph.js

```diff
@@ -31,7 +31,8 @@ function getGeometry(ctx, { scale, hScale, dataPoints }) {
   const width = ctx?.canvas.width - geo.scaleWidth;
   const height = ctx?.canvas.height - geo.scaleHeight;
 
-  geo.sectionWidth = (width - 2 * geo.graphPaddingH) / dataPoints;
+  // make sure geo.sectionWidth is not negative on really tiny screens
+  geo.sectionWidth = Math.abs((width - 2 * geo.graphPaddingH) / dataPoints);
```

Comment:
After loading the overview view I see an empty graph. It starts working when I change the timeline from 1h to 3h
<img width="404" alt="Screen Shot 2022-03-24 at 10 42 55" src="https://user-images.githubusercontent.com/20790348/159888694-d405d7b5-0d29-43a8-ac20-23028c695e03.png">

Reply:
Yeah, that was here before, it's not connected to Sentry issues.

Reply:
Then how it is working on dev?